Gemfile.lock

```diff
@@ -127,9 +130,10 @@ DEPENDENCIES
   rspec
   rubocop
   rubocop-rspec
+  simplecov
   sinatra
   sinatra-activerecord
   sinatra-contrib
 
 BUNDLED WITH
-   2.2.33
+   2.3.21
```

Comment:
@mrfamouskk7 why we changed bundler version?

Reply:
Changd it... actually my local bundler version was different compared to the one mentioned here... that's why it was updated